v22.2/create-and-configure-changefeeds.md

```diff
@@ -63,11 +63,17 @@ To create an {{ site.data.products.enterprise }} changefeed:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> CREATE CHANGEFEED FOR TABLE table_name, table_name2 INTO '{scheme}://{host}:{port}?{query_parameters}';
+CREATE CHANGEFEED FOR TABLE table_name, table_name2 INTO '{scheme}://{host}:{port}?{query_parameters}';
 ~~~
 
 {% include {{ page.version.version }}/cdc/url-encoding.md %}
 
+When you create a changefeed **without** specifying a sink, CockroachDB sends the changefeed events to the SQL client. Consider the following regarding the [display format](cockroach-sql.html#sql-flag-format) in your SQL client:
+
+- If you do not define a display format, the CockroachDB SQL client will automatically use `ndjson` format.
+- If you specify a format, the client will use that format (e.g., `--format=csv`).
+- If you set the client display format to `ndjson` and set the changefeed [`format`](create-changefeed.html#format) to `csv`, you'll receive JSON format with CSV nested inside. In the reverse situation, you'll receive a comma-separated list of JSON values.
```

Comment:
nit: would it make sense to break "X and inverse of X" into two separate bullet points? Perhaps:

> - If you set the client display format to `ndjson` and set the changefeed [`format`](create-changefeed.html#format) to `csv`, you'll receive JSON format with CSV nested inside.
> - If you set the client display format to `csv` and set the changefeed [`format`](create-changefeed.html#format) to `ndjson`, you'll receive a comma-separated list of JSON values.